test/autorest/azurespecialsgroup/zz_generated_header_client.go

```diff
@@ -86,6 +86,9 @@ func (client *HeaderClient) CustomNamedRequestIDHead(ctx context.Context, fooCli
 	if err != nil {
 		return HeaderClientCustomNamedRequestIDHeadResponse{}, err
 	}
+	if !runtime.HasStatusCode(resp, http.StatusOK, http.StatusNotFound) {
+		return HeaderClientCustomNamedRequestIDHeadResponse{}, runtime.NewResponseError(resp)
```

Comment:
Nit, HeaderCleintCustomRequestIDHeaderResponse... that's a mouthful :)

Reply:
This comes from the autorest test server swagger, the names aren't all that great.